docs/migrating-from-v2-to-v3.md

```diff
@@ -0,0 +1,15 @@
+# Migrating from V2 to V3
+
+Migrating from V2 to V3 is straightforward. The breaking changes that are required
```

Comment:
Migrating from previous versions to V3 is straightforward however there are two breaking changes in V3 that need to be addressed prior to migrating: